tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -28,16 +28,18 @@ sub quit_firefox {
 }
 
 sub run {
-    my ($self) = shift;
+    my ($self) = @_;
 
     # Define FIPS password for firefox, and it should be consisted by:
     # - at least 8 characters
     # - at least one upper case
     # - at least one non-alphabet-non-number character (like: @-.=%)
     my $fips_password = 'openqa@SUSE';
 
-    # Turn off screensaver before launch firefox in order to avoid the screensaver block
-    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
+    select_console 'root-console';
+    zypper_call 'in firefox';
```

Comment:
Is it necessary to install firefox? It is there by default for gnome desktop afaik.

Reply:
Yes, firefox is installed by default. Thanks for reminding and Updated.